pkg/taskutil/spec.go

```diff
@@ -89,15 +90,25 @@ type DefaultPlanOptions struct {
 	SpecURL string
 }
 
+func MetadataSpecURL() string {
+	if u := os.Getenv(MetadataAPIURLEnvName); u != "" {
+		return strings.Join([]string{strings.TrimSuffix(u, "/"), "spec"}, "/")
```

Comment:
Seems like something like `url.Parse` and/or `path.Join` would be more intuitive here?